keps/sig-storage/2485-read-write-once-pod-pv-access-mode/README.md

```diff
@@ -832,13 +948,31 @@ Try to be as paranoid as possible - e.g., what if some components will restart
 mid-rollout?
 -->
 
+Rolling out this feature involves enabling the ReadWriteOncePod feature gate
+across kube-apiserver, kube-scheduler, kubelet, and updating CSI driver and
+sidecar versions.
+
+The only way this rollout can fail is if a user does not update all components,
+in which case the feature will not work. See the above section on version skews
+for behavior in this scenario.
+
+Rolling out this feature does not impact any running workloads.
+
 ###### What specific metrics should inform a rollback?
 
 <!--
 What signals should users be paying attention to when the feature is young
 that might indicate a serious problem?
 -->
 
+If pods using ReadWriteOncePod PVCs fail to schedule, you may see an increase in
```

Comment:
If this enhancement is not enabled in the scheduler and the scheduler attempts to schedule a pod using ReadWriteOncePod PVCs, is it observed by this metric? I.e. if an upgrade is performed and this metric is elevated, what would a rollback look like to an observer of this metric?

Reply:
Yes, this metric will capture any failures in this scheduler plugin regardless of the feature gate.

This metric is a gauge that tracks the number of stuck pods. If an upgrade was performed and the metric was elevated, then rolling back would decrease the metric as the pods started to schedule again.